src/klein/test/__init__.py

```diff
@@ -0,0 +1,13 @@
+# -*- test-case-name: klein.test -*-
+# Copyright (c) 2017-2018. See LICENSE for details.
+
+"""
+Tests for L{klein}.
+"""
+
+from hypothesis import HealthCheck, settings
+
+settings.register_profile(
+    "patience", settings(suppress_health_check=[HealthCheck.too_slow])
+)
+settings.load_profile("patience")
```

Comment:
Moved here from `_strategies.py` below because Travis ran into "too slow" errors in `test_plating` and this will now prevent that as well.